src/contracts/interfaces/IIndexRegistry.sol

```diff
@@ -25,17 +25,17 @@ interface IIndexRegistry is IRegistry {
      * @param quorumNumbers is the quorum numbers the operator is registered for
      * @dev access restricted to the RegistryCoordinator
      */
-    function registerOperator(bytes32 operatorId, uint8[] memory quorumNumbers) external;
+    function registerOperator(bytes32 operatorId, bytes calldata quorumNumbers) external;
 
     /**
      * @notice Deregisters the operator with the specified `operatorId` for the quorums specified by `quorumBitmap`.
      * @param operatorId is the id of the operator that is being deregistered
      * @param quorumNumbers is the quorum numbers the operator is deregistered for
-     * @param quorumToOperatorListIndexes is an array of indexes for each quorum being removed from the quorumToOperatorList mapping
-     * @param globalOperatorListIndex is the index of the operator in the global operator list to be removed
+     * @param quorumToOperatorListIndexes is an array of indexes for each quorum as witnesses for the last operators to swap for each quorum
```

Comment:
I've harped on this language before but I think this is confusing. the 'as witnesses for' in particular could (IMO) at least use some elaboration.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/3fc195b23be66ac823f3e13550a45bf55ecc4887